plugins/filestorage/localfs/index.js

```diff
@@ -356,6 +356,10 @@ LocalFileStorage.prototype.createThumbnail = function (filePath, fileType, optio
   }
   var fileFormat = fileType.split('/')[1];
   fileType = fileType.split('/')[0];
+  // https://github.com/adaptlearning/adapt_authoring/issues/2065
+  if (fileFormat === 'svg+xml') {
```

Comment:
Can we get this check moved into the [isThumbnailTypeSupported](https://github.com/adaptlearning/adapt_authoring/blob/5cfae7a0ba0c0bccaa72fd719b71ff9271b31e8c/plugins/filestorage/localfs/index.js#L405-L414) method?